src/server/scripts/Outland/TempestKeep/Eye/boss_astromancer.cpp

```diff
@@ -146,11 +150,30 @@ struct boss_high_astromancer_solarian : public BossAI
         }).Schedule(52100ms, [this](TaskContext context)
         {
             me->SetReactState(REACT_PASSIVE);
-            Talk(SAY_SUMMON);
-            me->RemoveAllAuras();
-            me->SetModelVisible(false);
-            scheduler.DelayAll(21s);
-            scheduler.Schedule(6s, [this](TaskContext)
+            scheduler.DelayAll(22s);
+            // blink to room center in this line using SPELL_TELEPORT_START_POSITION and START_POSITION_X, START_POSITION_Y, START_POSITION_Z
+            scheduler.Schedule(1s, [this](TaskContext)
+            {
+                for (uint8 i = 0; i < 3; ++i)
+                {
+                    float o = rand_norm() * 2 * M_PI;
+                    if (i == 0)
+                    {
+                        me->SummonCreature(NPC_ASTROMANCER_SOLARIAN_SPOTLIGHT, CENTER_X + cos(o)*INNER_PORTAL_RADIUS, CENTER_Y + std::sin(o)*INNER_PORTAL_RADIUS, CENTER_Z, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 25000);
+                    }
+                    else
+                    {
+                        me->SummonCreature(NPC_ASTROMANCER_SOLARIAN_SPOTLIGHT, CENTER_X + cos(o)*OUTER_PORTAL_RADIUS, CENTER_Y + std::sin(o)*OUTER_PORTAL_RADIUS, PORTAL_Z, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 25000);
+                    }
+                }
+            }).scheduler.Schedule(2s, [this](TaskContext)
```

Comment:
don't need to repeat .scheduler, you are still invoking Schedule on the same scheduler